plugins/inlineStyles.js

```diff
@@ -1,24 +1,43 @@
 'use strict';
 
-const csstree = require('css-tree');
-const { querySelectorAll, closestByName } = require('../lib/xast.js');
-const cssTools = require('../lib/css-tools');
+/**
+ * @typedef {import('../lib/types').Specificity} Specificity
+ * @typedef {import('../lib/types').XastElement} XastElement
```

Comment:
i'm really starting to see the benefits of jsdocs inside a ts project. You can define your types outside, as plain ts files and use on js files :D